### PR TITLE
package standardization updates + postgres compat

### PR DESCRIPTION
**Pull request by fivetran-jamie**
**Are you a current Fivetran customer?** 

fivetran made PR

**What change(s) does this PR introduce?** 

- ensures postgres compatibility
- merged in old package update PRs (https://github.com/fivetran/dbt_shopify_source/pull/33) and added some tweaks

**Did you update the CHANGELOG?** 

- [ ] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [ ] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [ ] Yes, Issue/Feature [link bug/feature number here]
- [ ] No 

**How did you test the PR changes?** 

 

- [ ] CircleCi  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [ ] BigQuery
- [ ] Redshift
- [ ] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: models/tmp/stg_shopify__order_tmp.sql [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {{
     fivetran_utils.union_data(
-        table_identifier='order', 
+        table_identifier='order',
```
Comment (fivetran-jamie) [inline models/tmp/stg_shopify__order_tmp.sql:3]: @fivetran-reneeli actually does this need to be `shopify_order_identifier` ?

**Review by fivetran-reneeli — COMMENTED** [review]
Looks good! Nice job catching the `table_identifier` naming

File: models/tmp/stg_shopify__order_tmp.sql (hunk 1 above)
Comment (fivetran-jamie) [reply models/tmp/stg_shopify__order_tmp.sql:3]: sweet will also update

Comment (fivetran-jamie) [reply models/tmp/stg_shopify__order_tmp.sql:3]: wait wait i'm wrong!!!

Comment (fivetran-jamie) [reply models/tmp/stg_shopify__order_tmp.sql:3]: it should be the name of the source table node (which we ourselves have defined), _not_ the source table node's identifier

**Pull request merged**